lib/chef/provider/package/powershell.rb

```diff
@@ -36,7 +36,7 @@ def load_current_resource
 
         def define_resource_requirements
           super
-          if powershell_out("$PSVersionTable.PSVersion.Major").stdout.strip.to_i < 5
+          if node["languages"] && node["languages"]["powershell"] && node["languages"]["powershell"]["version"].strip.to_i < 5
```

Comment:
This can use the new powershell_version helper

```suggestion
          if powershell_version < 5
```